online-shopping-website/src/App.js

```diff
@@ -19,9 +18,6 @@ const Layout = () => {
                     <li>
                         <Link to="/">Products</Link>
                     </li>
-                    <li>
-                        <Link to="/searchbar">Search bar</Link>
```

Comment:
@emma-amar for navbar PR, remove entire `nav` tag body because links will be directly in the navbar instead.